### PR TITLE
On-ramp: fix default payment method selection

### PR DESCRIPTION
**Pull request by wachunei**
**Development & PR Process**
1. Follow MetaMask [Mobile Coding Standards](https://docs.google.com/document/d/1VJLwTRsUw_5EDq_o8d6sSbXUAYENLurkRitYO45eY5o/edit?usp=sharing)
2. Add `release-xx` label to identify the PR slated for a upcoming release (will be used in release discussion)
3. Add `needs-dev-review` label when work is completed
4. Add `needs-qa` label when dev review is completed
5. Add `QA Passed` label when QA has signed off

**Description**

This PR fixes a bug introduced by the new filter of customAction, the bug consisted in a wrongly selected default payment method.

**Screenshots/Recordings**

~~_If applicable, add screenshots and/or recordings to visualize the before and after of your change_~~

**Issue**

Progresses #???

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

### CODE REVIEW

File: app/components/UI/FiatOnRampAggregator/hooks/usePaymentMethods.ts [hunk 1]
```diff
@@ -44,17 +46,27 @@ function usePaymentMethods() {
...
+     */
+    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [
     error,
     isFetching,
     paymentMethods,
     sdk,
     selectedChainId,
-    selectedRegion?.id,
+    // selectedRegion?.id,
```
Comment (rickycodes) [inline app/components/UI/FiatOnRampAggregator/hooks/usePaymentMethods.ts:59]: why not jut delete?

**Review by rickycodes — APPROVED** [review]
just one question, but lgtm regardless

File: app/components/UI/FiatOnRampAggregator/hooks/usePaymentMethods.ts (hunk 1 above)
Comment (wachunei) [reply app/components/UI/FiatOnRampAggregator/hooks/usePaymentMethods.ts:59]: Discussed in slack
>I wanted to make it explicit that that is the only dependency that should be ignored and not others (sadly the eslint ignore is for the whole dependency list and not per dependcy)

**Review by andreahaku — APPROVED** [review]
LGTM

**Pull request merged**